src/main/java/scm/address/commons/core/GuiSettings.java

```diff
@@ -15,9 +15,13 @@ public class GuiSettings implements Serializable {
     private static final double DEFAULT_HEIGHT = 600;
     private static final double DEFAULT_WIDTH = 740;
 
+    private static final String THEME_LIGHT = "light";
+    private static final String THEME_DARK = "dark";
```

Comment:
I think it would be better if you use an interface with theme name and theme resource instead of using a String.

Reply:
Roger that! Will update!